### PR TITLE
Relax code guide rules about docstrings

### PR DESCRIPTION
**Pull request by pxeger**
https://chat.stackexchange.com/transcript/message/58909173#58909173

### CODE REVIEW

**Review by Lyxal — DISMISSED** [review]
This looks reasonable to me.

**Review by Lyxal — DISMISSED** [review]

**Comment by pxeger** [comment]
Sorry, I shouldn't really keep requesting your review; I should wait until I'm finished

**Comment by Lyxal** [comment]
> Sorry @Lyxal, I shouldn't really keep requesting your review; I should wait until I'm finished

It's only fair after I requested review from y'all so many times for #218 ;P

**Review by Lyxal — APPROVED** [review]
I'm good with this.

File: vyxal/structure.py [hunk 1]
```diff
@@ -9,9 +13,7 @@ def __init__(self, *branches: Branch):
         self.branches = branches
 
     def transpile(self) -> str:
```
Comment (ysthakur) [inline vyxal/structure.py:15]: I don't think we need this anymore, honestly. Transpilation should be separated from the definitions of the structures (and it already is anyway).

**Review by ysthakur — APPROVED** [review]
Looks good to me.

**Review by AMiller42 — APPROVED** [review]
ok

File: vyxal/structure.py (hunk 1 above)
Comment (pxeger) [reply vyxal/structure.py:15]: Yeah, I thought that too, but that's a PR for a different day I think

**Pull request merged**